src/App/Pages/Accounts/RegisterPageViewModel.cs

```diff
@@ -128,8 +154,15 @@ public async Task SubmitAsync(bool showLoading = true)
                     AppResources.AnErrorHasOccurred, AppResources.Ok);
                 return;
             }
-
-            // TODO: Password strength check?
+            if (PasswordStrengthViewModel.PasswordStrengthLevel == PasswordStrengthLevel.Weak)
+            {
```

Comment:
You'll have to change it to `PasswordStrengthViewModel.PasswordStrengthLevel <= PasswordStrengthLevel.Weak ` when you add the new enum value